goboy/emu.go

```diff
@@ -1,9 +1,18 @@
 package main
 
+import "fmt"
+
 func (g *GameBoy) Step() {
 	/* 3 is the max length of an instruction (I think) */
 	pc := g.regs[PC]
 	opCode := g.mainMemory.read(pc)
+	fmt.Printf("PC: 0x%04x\n", pc)
```

Comment:
Ultimately these should go somewhere else. This is purely for debugging right?

Reply:
correct. I don't intend for this to be the final PR either